.palantir/revapi.yml

```diff
@@ -261,6 +266,17 @@ acceptedBreaks:
         \ T) throws java.io.IOException, com.fasterxml.jackson.core.JacksonException\
         \ @ org.apache.iceberg.rest.RESTSerializers.UpdateRequirementDeserializer"
       justification: "False positive - JacksonException is a subclass of IOException"
+    - code: "java.method.numberOfParametersChanged"
+      old: "method void org.apache.iceberg.SetLocation::<init>(org.apache.iceberg.TableOperations)"
+      new: "method void org.apache.iceberg.SetLocation::<init>(java.lang.String, org.apache.iceberg.TableOperations)"
+      justification: "SetLocation should only be constructed by Iceberg code. Hence\
+        \ the change of constructor params shouldn't affect users."
+    - code: "java.method.numberOfParametersChanged"
+      old: "method void org.apache.iceberg.SetStatistics::<init>(org.apache.iceberg.TableOperations)"
+      new: "method void org.apache.iceberg.SetStatistics::<init>(java.lang.String,\
+        \ org.apache.iceberg.TableOperations)"
+      justification: "SetStatistics should only be constructed by Iceberg code. Hence\
+        \ the change of constructor params shouldn't affect users."
```

Comment:
There should not be revapi exceptions. If you add to an interface, then add a default implementation.